.bazelci/presubmit.yml

```diff
@@ -21,26 +21,6 @@ tasks:
       # These tests are currently incompatible with RBE
       - "-//tests/integration/override_targets"
       - "-//tests/com/github/bazelbuild/rules_jvm_external/maven:OutdatedTest"
-  ubuntu1804_4_0_0:
-    platform: ubuntu1804
-    bazel: 4.0.0
```

Comment:
Dropping support for Bazel 4 is on the roadmap, but I'm curious about why this change is made in this PR